.github/workflows/trigger-ui-tests.yml

```
@@ -24,7 +24,14 @@ jobs:
 
           git fetch --prune origin
           BRANCHES=$(git ls-remote --heads origin | awk -F'/' '{print $3"/"$4}' | grep '^customer/')
+          SKIP_BRANCHES=("customer/sav" "customer/vsb-tuo" "customer/zcu-data")
+          
           for branch in $(echo "$BRANCHES" | sed -e 's/[\[\]"]//g' -e 's/,/\n/g'); do
+            if [[ " ${SKIP_BRANCHES[@]} " =~ " ${branch} " ]]; then
+              echo "Skipping branch $branch"
+              continue
+            fi
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)